index.bs

```diff
@@ -269,11 +315,11 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
     <h3 id="allowlists">Allowlists</h3>
     <p>A permissions policy <dfn export
     lt="allowlist|allowlists">allowlist</dfn> is conceptually a set of
-    [=origins=]. An <a>allowlist</a> may be either:
+    <a>permissions-source-expression</a>s. An <a>allowlist</a> may be either:
```

Comment:
I would still maintain that it is a set of origins, conceptually. It's all of the origins (potentially infinite) which match the various patterns used in the header or attribute. The match algorithm is essentially testing set inclusion.

Structurally, it is definitely a set of expressions, as you mention below. (This is all pretty pedantic, though)